### ISSUE TITLE
[Feature Reuest] Make this Hassio Addon

### ISSUE THREAD

**Issue by thundergreen** [JarvyJ/HomeIntent#257]
Hello, thanks first for this awesome masterpiece. I struggled hard with intents and did not know really how to start. Then I found your App and set up everything in 5 minutes and EVERYTHING works well. I just need to adjust rhasspy for better understanding.

As I am running Hassos and therefor no Custom Dockers  to run, I thought it might be just awesome to make an Addon? What you think?

Cheers

Thunder

**Comment by JarvyJ** [comment]
Yes! This is a great idea and part of the plan. Once I properly support satellites, I am going to make it available as an addon. I figured I'd wait, since folks will probably need satellites with a Hass addon.

**Comment by thundergreen** [comment]
Good point! Absolutely great. Waiting the outcome. For the time being I use HomeIntent apart from Home assistant connected via MQTT and Stellites :) 

Thanks so mich for this awesome masterpiece!

**Comment by JarvyJ** [comment]
You're welcome! Glad you're enjoying it!

**Comment by SamJongenelen** [comment]
Great! Subscribing
